rpcq/_client.py

```diff
@@ -27,19 +27,20 @@
 
 _log = logging.getLogger(__name__)
 
+DEFAULT_TIMEOUT = 20_000  # ms
```

Comment:
This was well justified for the USRPs but I'm not sure what the generic justification is for 20 seconds.

Reply:
Obsolete since removing default timeout anyways...